src/vm/arm/stubs.cpp

```diff
@@ -534,13 +534,21 @@ void LazyMachState::unwindLazyState(LazyMachState* baseState,
 
     do
     {
+#ifdef DACCESS_COMPILE
```

Comment:
This change is necessary for FEATURE_PAL only.The Thread::VirtualUnwindCallFrame already does this internally.